Tasks/AzureRmWebAppDeploymentV3/make.json

```diff
@@ -6,6 +6,11 @@
                 "url": "https://vstsagenttools.blob.core.windows.net/tools/MSDeploy/3.6/MSDeploy.zip",
                 "dest": "./"
             },
+            {
+                "archiveName": "MSDeploy.zip",
+                "url": "https://vstsagenttools.blob.core.windows.net/tools/MSDeploy/3.6/MSDeploy.zip",
+                "dest": "./webdeployment-common/"
+            },
```

Comment:
I did not get your description. Why we need above line here where destination is ./ , it will be under webdeployment-common folder only, right ?